.changes/unreleased/tinymce-TINY-11707-2025-01-07.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Added
+body: '`link_attributes_postprocess` option that allows overriding attributes of a
+  link that would be inserted through the link dialog.'
```

I think it looks a bit weird if we just have the backticks at the beginning of the sentence. We used `New` before.

```suggestion
body: 'New `link_attributes_postprocess` option that allows overriding attributes of a
  link that would be inserted through the link dialog.'
```